### DISCUSSION TITLE
Metrics are not pulling in Mirth Types

### DISCUSSION THREAD

**Discussion by AnimalMind** [waytohealth/mirth_channel_exporter#8] [General]
Hello and thank you so much for creating this. I am able to pull the metrics into Prometheus and show them via grafana but what I noticed is that only the mirth_up stat is coming through an not the custom ones in metrics.go file. 

I have this loaded using docker compose. Thanks again.

**Comment by AnimalMind** [comment]
On the terminal view of the docker I am getting: `expected element type <list> but have <html>`

**Reply by mkopinsky** [reply]
Is it returning `mirth_up 0` or `mirth_up 1`? If it can't reach Mirth it indeed won't be able to return the other metrics.

That error indicates to me that either it is hitting something other than Mirth, or it is not successfully authenticating against the API.

**Reply by AnimalMind** [reply]
You are correct its returning 0 for mirth up. I am not sure why I have the url correct and I am able to ping it from within the terminal and access it locally through a browser.

**Reply by AnimalMind** [reply]
What interesting is that its just returning: 2022/08/02 16:27:14 expected element type <list> but have <html>
